docs/global-content-repository.md

```diff
@@ -0,0 +1,49 @@
+# Global Content Repository
+
+Altis includes a Global Content Repository framework. This is a site on the network that can be used to distribute content elsewhere via the REST API or directly to other sites on the network.
+
+By default the Global Content Repository only exposes the user management admin. Other Altis modules may extend this, for example [the Altis Media module has a Global Media Library feature](docs://media/global-media-library.md) built on top of the Global Content Repository that allows for uploading media files and accessing them from all sites on the network.
+
+## User Management
+
+Users on the multisite network can have different roles on different sites. This gives you control over who can add and edit global content versus who can only use the global content on their own site.
+
+You might have a user who is an author on the main site, but only a subscriber to the Global Content Repository meaning they could read content to use in their posts but not create or edit global content.
```

Comment:
They would be able to read media attachments even without being logged in, those are public.

Reply:
Roles and capabilities are confusing and I don't know that the example helps that much. A user who has a subscriber role on the global content repository site cannot read images without changing the permissions. If this is the intent, we need to build the functionality/change the caps associated with subscribers first. 

Testing locally, I added a subscriber to the global content site and this is what I get when I'm logged in as that user:

![Screen Shot 2021-05-04 at 8 38 15 AM](https://user-images.githubusercontent.com/991511/117021190-5bd97200-acb4-11eb-9f8f-de18495717db.png)

Reply:
I was thinking in terms of the REST API here but on the global site admin they wouldn't see the media library so you're right about that. The above would be more accurate if it the user had no role on the global site.

I wasn't thinking about the Editor actually logging into the global site admin in this case.

I need to think on that some more, whether all users should have a role on the global site or just assume or just handle adding it when they do something like upload an image...

Reply:
Yeah the language makes it sound like an _actual_ subscriber (e.g. user with that role). I will think about it as well and see if I can think of a better way to describe it. 

I think handling when they upload an image probably makes the most sense, otherwise we're potentially migrating a bunch of users that possibly (probably) don't really care.

Reply:
Updated the language to suggest what happens when there is no role on either site rather than mentioning subscribers. I think as this feature develops we'll get a clearer idea of what we need to document better or build out handling for in the generic sense.